VideoWeb/VideoWeb/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -4330,6 +4330,7 @@ export class ParticipantForUserResponse implements IParticipantForUserResponse {
     hearing_role?: string | undefined;
     current_room?: RoomSummaryResponse | undefined;
     linked_participants?: LinkedParticipantResponse[] | undefined;
+    hasInterpreterLink?: boolean | undefined;
```

Comment:
this is an auto-generated file. Your change will be lost the next time someone runs the update command. Seems easy enough to wrap into a function

Reply:
ok